src/pybind/mgr/dashboard/tests/__init__.py

```diff
@@ -334,3 +337,56 @@ def get_settings(cls):
             'RGW_API_SECRET_KEY': 'fake-secret-key',
         }
         mgr.get_module_option = Mock(side_effect=settings.get)
+
+
+# pylint: disable=protected-access
+class Waiter(threading.Thread):
+    def __init__(self, task_name, task_metadata, tc, version):
+        super(Waiter, self).__init__()
```

Comment:
The conflict doesn't seem to be resolved here properly @pereman2

Reply:
This whole `class Waiter` block can be removed, the main PR didn't involve a change to this. `patch_orch` is expected tho.

Reply:
done, @aaSharma14 @pereman2